latch/ldata/_transfer/upload.py

```diff
@@ -102,7 +102,7 @@ def upload(
         num_bars = get_max_workers()
         show_total_progress = True
 
-    with ProcessPoolExecutor(max_workers=get_max_workers()) as exec:
+    with ProcessPoolExecutor(max_workers=(get_max_workers())) as exec:
```

Comment:
I added an arg called cores to sync which makes it so we use fewer cores -- honestly our max workers might be too much because I have noticed that it gets stuck in the multiprocessing library (just kinda peters out and stops doing anything).